### PR TITLE
Grammar changes

### PR DESCRIPTION
**Pull request by mirakst**
- Fixed negative number handling, now correctly parses `2--2` as 2 - (-2)
- Removed switch from grammar
- Continue and break are now only allowed inside if-statements in loop constructs
- Added a few new example programs
- Removed switch from example program cm_to_inch_converter

### CODE REVIEW

File: Example programs/cm_to_inch_converter.step [hunk 1]
```diff
@@ -10,18 +10,6 @@ setup
...
-        ##when 1 do
-
-        when 2 do
-            fallthrough
-
-
-        when 3 do
-            ass(b)
-
-        otherwise do##end switch
```
Comment (KarmaKamikaze) [inline Example programs/cm_to_inch_converter.step:23]: Rest in peace, sweet prince 🌹

**Review by KarmaKamikaze — APPROVED** [review]
image

**Review by Sarmisuper — APPROVED** [review]
lgtm
image

File: Example programs/cm_to_inch_converter.step (hunk 1 above)
Comment (mirakst) [reply Example programs/cm_to_inch_converter.step:23]: 😶‍🌫️

**Pull request merged**